packages/server/tests/UserProfileInformation.test.ts

```diff
@@ -31,7 +33,7 @@ test.group('Users Profile information tests', (group) => {
         });
 
         const { body: rawBody } = await supertest(BASE_URL)
-            .post('/user/profile-information')
+            .post(urlcat(USER_ROUTES_GROUP_PREFIX, 'profile-information'))
```

Comment:
I do not really have a strong opinion on this, but using `USER_ROUTES_GROUP_PREFIX` here breaks black-box testing principles.

Reply:
Yes you're right :thinking: 
We were already doing the same for both MPE and MTV PREFIX inside others tests
I don't really know if we came to a conclusion on them ?
We could duplicate them inside tests utils ?
Gonna forward an implem suggestion